src/adapters/tcp.rs

```diff
@@ -27,15 +31,30 @@ pub const INPUT_BUFFER_SIZE: usize = u16::MAX as usize; // 2^16 - 1
 
 #[derive(Clone, Debug, Default)]
 pub struct TcpConnectConfig {
+    bind_device: Option<String>,
+    source_address: Option<SocketAddr>,
     keepalive: Option<TcpKeepalive>,
 }
 
 impl TcpConnectConfig {
+    /// Bind the TCP connection to a specific interface, identified by its name. On other systems
+    /// this opion will be ignored.
```

Comment:
```suggestion
    /// Bind the TCP connection to a specific interface, identified by its name. 
    /// This option works in Unix, on other systems, it will be ignored.
```

Reply:
oh yes, I changed that sentence multiple times. At some point I must have dropped the Unix part accidentally .